terminologies/metaDataModelType/commonDataElements.jsonld

```diff
@@ -2,7 +2,7 @@
   "@id": "https://openminds.ebrains.eu/controlledTerms/metaDataModelType/commonDataElements",
   "@type": "https://openminds.ebrains.eu/controlledTerms/MetaDataModelType",
   "name": "common data elements",
-  "definition": null,
+  "definition": "Common Data Element (CDE) define standardized key terms or concepts for describing clinical research or studies in order to enhance data quality and cross validations or visualizations.",
```

Comment:
"Common Data Element**S** (CDEs) define..." (typo?)

Reply:
changed it to Common Data Element (CDE) literature is a bit confusing on the matter... 

sometimes only Common Data Element with abbreviation CDE is used, sometimes it's fully written as Common Data Elements, but still abbreviated as CDE, and sometimes it's fully written as Common Data Elements and abbreviated as CDEs... (I suggest to stick with the fully singular form for now; what do you think?)

Reply:
Okay, but do you want to also change filename, `@id` and name then? All have the plural s at the moment.

Reply:
mh... I thought I did. at least for the `@id`. must have not saved it or just forgotten about it while thinking it :see_no_evil: I'll change it right away

Reply:
I forgot the filename and changed it now. `@id` was already changed.